### PR TITLE
Document running Jupyter notebooks via `opensafely`

### PR DESCRIPTION
**Pull request by StevenMaude**
Document that you can run JupyterLab via the `opensafely` command-line tool.

### CODE REVIEW

File: docs/opensafely-cli.md [hunk 1]
```diff
@@ -125,3 +125,21 @@ To pull the most recent Docker images to your machine, run:
...
+environments for developing code.
+
+You can run JupyterLab to use Jupyter notebooks via the `opensafely`
+tool. This ensures that the Python code you write will work in the
+OpenSAFELY environment.
+
+To use Jupyter notebooks via `opensafely`:
+
+1. Run `opensafely jupyter` to start a JupyterLab server on your
+   computer.
```
Comment (bloodearnest) [inline docs/opensafely-cli.md:141]: add "in the directory containing the code you would like to run with jupyter" ?

File: docs/opensafely-cli.md [hunk 2]
```diff
@@ -125,3 +125,21 @@ To pull the most recent Docker images to your machine, run:
...
+2. You will see long URLs starting `http://localhost…` and
```
Comment (bloodearnest) [inline docs/opensafely-cli.md:142]: It should automatically open a brower

**Review by bloodearnest — CHANGES_REQUESTED** [review]

File: docs/getting-started.md [hunk 3]
```diff
@@ -773,6 +773,8 @@ against real data.  [Read more about permissions here](permissions.md).
 In the meantime, take a look at the rest of our documentation for more
 detail on the subjects covered in this tutorial. For example:
 
+* There is a [more complete guide](opensafely-cli.md) to the OpenSAFELY
```
Comment (madwort) [inline docs/getting-started.md:776]: as part of the accessibility guidelines, I think we need to make sure we're using meaningful link text - I don't think `[more complete guide] to the cli` stands on its own well enough as link text, how about `more complete [guide to the cli]`?

**Review by madwort — CHANGES_REQUESTED** [review]
nitpick link text, otherwise looks good

File: docs/opensafely-cli.md (hunk 2 above)
Comment (StevenMaude) [reply docs/opensafely-cli.md:142]: I'll mention that as the default, as it seems like it should be the case.

(This isn't the behaviour for me, and @iaindillingham said [similar](https://github.com/opensafely/documentation/discussions/530). This is why I specified the copy-paste step explicitly.)

File: docs/getting-started.md (hunk 3 above)
Comment (StevenMaude) [reply docs/getting-started.md]: Yep, agree!

**Comment by bloodearnest** [comment]
Added a minor suggestion.

I don't want to land this until I've finished the testing I'm doing with internal users, as there's still some issues for some folks.

It's very difficult to test this feature, so my plan was to release quietly, having trusted users test on their machines, and then announcing in docs once it's more solid.

**Review by madwort — APPROVED** [review]
wfm

**Comment by StevenMaude** [comment]
@bloodearnest: sounds good!

**Review by bloodearnest — APPROVED** [review]

**Comment by bloodearnest** [comment]
Right, the command seems to be broadly working, so good to go from my pov

**Pull request merged**